recipes/pokrok/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 John Didion <github@didion.net>
```

Comment:
I've put in a PR (https://github.com/jdidion/pokrok/pull/2) to include the license file in the source distribution for future releases. If the PR is accepted and a new release is spun out, please remove this license file when the feedstock gets updated.